src/plugins/intel_gpu/src/kernel_selector/kernels/rms/rms_kernel_bfyx_opt.cpp

```diff
@@ -8,6 +8,24 @@
 
 namespace kernel_selector {
 static constexpr size_t subgroup_size = 16;
+static constexpr size_t max_items_num = 8;
+
+// Compute maximum possible LWS that does not exceed device capabilities and optimizes number of global memory reads
+std::pair<int, int> get_item_num_and_lws(const rms_params params, size_t data_size);
```

Comment:
Nitpick: You don't need this prototype declaration

Reply:
It will raise "-Werror=missing-declarations" in compiling if I didn't declare this.

Reply:
You just need to make it either static or put to anonymous namespace to avoid that

Reply:
done